modules/standard/Random.chpl

```diff
@@ -798,6 +806,31 @@ module Random {
         PCGRandomStreamPrivate_count = 1;
       }
 
+      /*
+        Constructs a new stream of random numbers using the specified seed
+        and parallel safety.
+
+        :arg eltType: The element type to be generated.
+        :type eltType: `type`
+
+        :arg seed: The seed to use for the PRNG.  Defaults to `oddCurrentTime` from
+         :type:`RandomSupport.SeedGenerator`.
+        :type seed: `int(64)`
+
+        :arg parSafe: The parallel safety setting.  Defaults to `true`.
+        :type parSafe: `bool`
+
+        :returns: an owned PCGRandomStream
+      */
```

Comment:
If we keep this change, the documentation & method should also appear in RandomStreamInterface